src/core/crash-handler.c

```diff
@@ -80,10 +79,7 @@ _noreturn_ static void crash(int sig, siginfo_t *siginfo, void *context) {
                         (void) chdir("/");
 
                         /* Raise the signal again */
-                        pid = raw_getpid();
-                        if (rt_tgsigqueueinfo(pid, gettid(), sig, siginfo) < 0)
-                                (void) kill(pid, sig); /* raise() would kill the parent */
```

Comment:
`kill()` was used but now `raise()` is used. If it is intended, then please comment that in the commit message.